easybuild/toolchains/fft/intelfftw.py

```diff
@@ -85,7 +85,7 @@ def _set_fftw_variables(self):
         self.log.debug('fftw_libs %s' % fftw_libs.__repr__())
 
         self.FFT_LIB_DIR = self.BLAS_LIB_DIR
-        self.FFT_INCLUDE_DIR = self.BLAS_INCLUDE_DIR
+        self.FFT_INCLUDE_DIR = os.path.join(self.BLAS_INCLUDE_DIR, 'fftw')
```

Comment:
@akesandgren Ah, `BLAS_INCLUDE_DIR` is a list, I overlooked that.

So, should become:

```python
self.FFT_INCLUDE_DIR = [os.path.join(d, 'fftw') for d in self. BLAS_INCLUDE_DIR]